### ISSUE TITLE
Stored Object: paper:openreview.VETJKjmyJyK

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed#5827]
{
  "sourceId": "openreview",
  "paperId": "VETJKjmyJyK",
  "url": "https://openreview.net/forum?id=VETJKjmyJyK",
  "title": "Embedding Transfer via Smooth Contrastive Loss\ud83d\udcdd",
  "authors": "Sungyeon Kim, Dongwon Kim, Minsu Cho, Suha Kwak",
  "abstract": "This paper presents a novel method for embedding transfer, a task of transferring knowledge of a learned embedding model to another. Our method exploits pairwise similarities between samples in the source embedding space as the knowledge, and transfers it through a loss function used for learning target embedding models. To this end, we design a new loss called smooth contrastive loss, which pulls together or pushes apart a pair of samples in a target embedding space with strength determined by their semantic similarity in the source embedding space; an analysis of the loss reveals that this property enables more important pairs to contribute more to learning the target embedding space. Experiments on metric learning benchmarks demonstrate that our method improves performance, or reduces sizes and embedding dimensions of target models effectively. Moreover, we show that deep networks trained in a self-supervised manner can be further enhanced by our method with no additional supervision. In all the experiments, our method clearly outperforms existing embedding transfer techniques.",
  "timestamp": "2025-07-09T05:23:51.706Z",
  "rating": "novote",
  "publishedDate": "",
  "tags": [
    "embedding transfer",
    "knowledge distillation",
    "deep metric learning",
    "representation learning"
  ],
  "doi": "",
  "journalName": "",
  "sourceType": "url"
}

**Comment by dmarx** [comment]
{
  "_data": {
    "sourceId": "openreview",
    "paperId": "VETJKjmyJyK",
    "url": "https://openreview.net/forum?id=VETJKjmyJyK",
    "title": "Embedding Transfer via Smooth Contrastive Loss📝",
    "authors": "Sungyeon Kim, Dongwon Kim, Minsu Cho, Suha Kwak",
    "abstract": "This paper presents a novel method for embedding transfer, a task of transferring knowledge of a learned embedding model to another. Our method exploits pairwise similarities between samples in the source embedding space as the knowledge, and transfers it through a loss function used for learning target embedding models. To this end, we design a new loss called smooth contrastive loss, which pulls together or pushes apart a pair of samples in a target embedding space with strength determined by their semantic similarity in the source embedding space; an analysis of the loss reveals that this property enables more important pairs to contribute more to learning the target embedding space. Experiments on metric learning benchmarks demonstrate that our method improves performance, or reduces sizes and embedding dimensions of target models effectively. Moreover, we show that deep networks trained in a self-supervised manner can be further enhanced by our method with no additional supervision. In all the experiments, our method clearly outperforms existing embedding transfer techniques.",
    "timestamp": "2025-07-09T05:23:51.706Z",
    "rating": "novote",
    "publishedDate": "",
    "tags": [
      "embedding transfer",
      "knowledge distillation",
      "deep metric learning",
      "representation learning"
    ],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2025-07-09T05:23:52.483Z",
    "update_mode": "append",
    "issue_number": 5827
  },
  "type": "initial_state"
}
